src/tests/Microsoft.PowerFx.Connectors.Tests/ODataDelegationTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Threading.Tasks;
+using System.Web;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.Tests;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Connectors.Tests
+{
+    public class ODataDelegationTests : PowerFxTest
+    {
+        [Theory]
+        [InlineData("Filter(Table, x > 10)", "https://contoso.com/api/list?$filter=x+gt+10")]
+        [InlineData("Filter(Table, x > 10 + 5)", "https://contoso.com/api/list?$filter=x+gt+15")]
+        [InlineData("Filter(Table, x > 2 * myNumber)", "https://contoso.com/api/list?$filter=x+gt+100")]
```

Comment:
> x > 2 * myNumber [](http://example.com/codeflow?start=34&length=17)

This is a really great test - illustrating the cross over between OData filter and interpreter.